config/initializers/flipper.rb

```diff
@@ -1,9 +1,12 @@
 require "flipper"
 require "flipper/adapters/active_record"
+require "flipper/instrumentation/log_subscriber"
 
 Flipper.configure do |config|
   config.default do
-    Flipper.new(Flipper::Adapters::ActiveRecord.new)
+    adapter = Flipper::Adapters::ActiveRecord.new
+    instrumented = Flipper::Adapters::Instrumented.new(adapter, :instrumenter => ActiveSupport::Notifications)
+    Flipper.new(instrumented, :instrumenter => ActiveSupport::Notifications)
```

Comment:
I know I'm annoying but while we don't enforce a style with Rubocop it'd be better to stick to symbol syntax :sweat: . The reason hash rockets are used here is because that code is from 2013 :sweat_smile:

Reply:
You're totally right. I've copy/pasted from `Flipper`... 
I'll changed it.

Reply:
Done.